src/Microsoft.Azure.WebJobs.Host/Bindings/AttributeConverter.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    internal class AttributeCloner
```

Comment:
I blame StyleCop. 
We already have a AttributeClone<T> and AttributeClone.
StyleCop won't allow both in the same file AttributeCloner.cs.

---
In reply to: [110198953](https://github.com/Azure/azure-webjobs-sdk/pull/1093#discussion_r110198953) [](ancestors = 110198953)